_posts/en/newsletters/2022-02-09-newsletter.md

```diff
@@ -0,0 +1,114 @@
+---
+title: 'Bitcoin Optech Newsletter #186'
+permalink: /en/newsletters/2022/02/09/
+name: 2022-02-09-newsletter
+slug: 2022-02-09-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a discussion about changing relay
+policy for replace-by-fee transactions and includes our regular sections
+with the summary of a Bitcoin Core PR Review Club meeting, announcements
+of new releases and release candidates, and descriptions of notable
+changes to popular Bitcoin infrastructure projects.
+
+## News
+
+- **Discussion about RBF policy:** Gloria Zhao started a
+  [discussion][zhao rbf] on the Bitcoin-Dev mailing list about
+  Replace-by-Fee ([RBF][topic rbf]) policy.  Her initial email provides
```

Comment:
maybe replace "initial" with "well received" as it seems a lot of people have been very delighted about the mail?